src/views/GovernanceReviewTeam/Actions/EmailRecipientsFields.tsx

```diff
@@ -62,46 +149,46 @@ export default ({
   const flags = useFlags();
 
   // Contacts query
-  const { contacts } = useSystemIntakeContacts(systemIntakeId);
-  // Get system intake from Apollo cache
-  const { systemIntake } = useSystemIntake(systemIntakeId);
-
-  // Get action type from path
-  const defaultRecipientsCount = useRef(
-    recipients.shouldNotifyITInvestment ? 3 : 2
-  ).current;
-
-  const requester = {
-    euaUserId: systemIntake!.euaUserId,
-    commonName: systemIntake!.requester.name,
-    component: systemIntake!.requester.component!,
-    email: systemIntake!.requester.email || '',
-    role: 'Requester'
-  };
+  const { contacts, createContact } = useSystemIntakeContacts(systemIntakeId);
+  const {
+    requester,
+    businessOwner,
+    productManager,
+    isso,
+    additionalContacts
+  } = contacts.data;
+
+  const defaultRecipients = useRef(recipients).current;
 
   /** Formatted array of contacts in order of display */
-  const contactsObject = contacts
-    ? [
-        requester,
-        contacts.businessOwner,
-        contacts.productManager,
-        ...(contacts.isso.commonName ? [contacts.isso] : []), // Only include ISSO if present
-        contacts.additionalContacts
-      ].flat()
-    : [];
-
-  // Number of possible recipients
-  const contactsCount = contactsObject.length + 2;
-  // Number of selected contacts
-  const selectedContacts = contactsObject.filter(({ email }) =>
+  const contactsArray = [
+    requester,
+    businessOwner,
+    productManager,
+    ...(isso.commonName ? [isso] : []), // Only include ISSO if present
+    additionalContacts
+  ].flat();
+
+  const unverifiedRecipients = contactsArray.filter(({ id }) => !id);
+
+  // Default recipients count
+  const defaultRecipientsCount =
+    (defaultRecipients.shouldNotifyITInvestment ? 3 : 2) +
+    unverifiedRecipients.filter(({ role }) => role !== 'Requester').length;
+  // Possible recipients count
+  const contactsCount = contactsArray.length + 2;
+  // Selected recipients count
+  const selectedContacts = contactsArray.filter(({ email }) =>
     recipients.regularRecipientEmails.includes(email!)
   ).length;
   const selectedCount =
     selectedContacts +
     Number(recipients.shouldNotifyITGovernance) +
     Number(recipients.shouldNotifyITInvestment);
 
```

Comment:
This value is not being rerendered when updated/check/uncheck contacts

Reply:
I wasn't able to reproduce these numbers not updating... Screenshot below is updated numbers after verifying an existing contact and adding a new one.

![Screen Shot 2022-10-12 at 3 59 38 PM](https://user-images.githubusercontent.com/60187543/195651146-056bf90c-d86c-4dde-a423-a6bba9c47318.png)

Were you testing a specific action or were they not updating for any of them? And were you logged in locally or with your EUA?